Sources/RepresentableKit/UIViewAdaptor.swift

```diff
@@ -86,17 +88,14 @@ struct UIViewHolder<Content: UIView> {
     
     let idealSizeCalculator: UIViewIdealSizeCalculator<Content>
     
-    var idealSize = IdealSize(width: nil, height: nil)
-        
-    mutating func updateView(_ uiView: Content) {
+    func updateView(_ uiView: Content) {
         storage.view = uiView
         uiView.apply(flexibility: flexibility)
-        updateSize(proxySize: uiView.frame.size)
     }
     
-    mutating func updateSize(proxySize: CGSize) {
-        guard let uiView = storage.view else { return }
-        idealSize = idealSizeCalculator.viewIdealSizeInSize(uiView, proxySize)
+    func idealSize(proxySize: CGSize) -> IdealSize? {
```

Comment:
Seems like we can make this private?

Reply:
We can't since this function is called from outside this struct. Internal should be good enough IMO.